pages/_app.js

```diff
@@ -1,27 +1,30 @@
-import Layer0RUM from 'rum'
 import '@/styles/globals.css'
 import { useEffect } from 'react'
+import Layer0RUM from '@/layer0/rum'
```

Comment:
typo? not `@layer0/rum`?

Reply:
Nope, all the Layer0 specific files have been moved into a layer0 folder. This folder as mentioned in jsconfig,json can be accessed directly via @/layer0 prefix.